clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -635,23 +636,33 @@ public void testQuotaMetrics() throws Exception {
         assertEquals(300, maxMetric.value(), EPSILON);
     }
 
+    @Test
+    public void testGetOffsetsByTimes() {
```

Comment:
I don't think so. It seems better to have more testing closer to the source since 1) it's cheaper, and 2) it is easier to locate bugs when the tests can isolate functionality.